components/toast_wrapper/toast_wrapper.jsx

```diff
@@ -254,6 +263,18 @@ class ToastWrapper extends React.PureComponent {
         );
     }
 
+    getSearchHintToastText = () => {
+        return (
+            <FormattedMessage
+                id='postlist.toast.searchHint'
```

Comment:
Heads up that you need to add that translation to `i18n/en.json`. You can use `make i18n-extract` for that. See https://developers.mattermost.com/contribute/webapp/using-i18n-extract/ for more information.

Reply:
Oh, sorry for this. Done.